src/main/kotlin/com/rose/gateway/config/schema/ChatConfig.kt

```diff
@@ -12,27 +13,36 @@ import kotlinx.serialization.Serializable
  * @constructor Creates a "chat config" with the provided data
  *
  * @param enabled Whether the extension is enabled
+ * @property showRoleColor Whether discord role colors are shown
```

Comment:
Move this above @constructor on line 13 for consistency. Also, capitalize to Discord